UI/forms/OBSBasicSettings.ui

```diff
@@ -3735,6 +3744,59 @@
          </item>
         </layout>
        </widget>
+
+       <widget class="QWidget" name="screenshotsPage">
+        <layout class="QFormLayout" name="formLayout_1003">
+         <property name="fieldGrowthPolicy">
+          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
+         </property>
+         <property name="labelAlignment">
+          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+         </property>
+         <item row="0" column="0">
+          <widget class="QLabel" name="label_1008">
+           <property name="minimumSize">
+            <size>
+             <width>170</width>
+             <height>0</height>
+            </size>
+           </property>
+           <property name="text">
+            <string>Basic.Settings.ScreenShots.SavePath</string>
+           </property>
+           <property name="alignment">
+            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+           </property>
+           <property name="buddy">
+            <cstring>screenshotOutputPath</cstring>
+           </property>
+          </widget>
+         </item>
+                <item row="0" column="1">
```

Comment:
The formatting below (and including) this line is kind of weird; reformat it to better match the other lines?